handler.py

```diff
@@ -25,24 +32,31 @@ def main(event, _):
     rialto_topic_arn = os.getenv('RIALTO_TOPIC_ARN', "rialto")
     aws_region = os.getenv('AWS_REGION', "us-west-2")
 
-    request_content_type = event['headers']['Content-Type']  # capture this value for use throughout
+    request_body = event['body']
+    request_content_type = event['headers']['Content-Type']
+
+    # Verify that the body is properly encoded and has a supported content type
+    # BEFORE setting up AWS resources
+    verify_query = is_malformed_query(request_body, request_content_type)
```

Comment:
Why test this here? Won’t Neptune basically repeat the same?

Reply:
It should, but if you recall, for certain malformed queries, neptune barfs. :-)

Reply:
Note: We were doing this in Go for this reason: https://github.com/sul-dlss-labs/sparql-loader/blob/a9e21d14f00462bc63300ac386a526384bb36250/runtime/handler.go#L46

Reply:
Is it terrible for us to just let neptune barf?  If we truly want this to be an invisible proxy, that's the expected behavior.  Furthermore, we don't want to slow this loader down any more than necessary.

Reply:
I'm OK ignoring it as long as we know that may be a source. With ETL I doubt it will happen, this is likely more of a concern when/if anyone else hit's sparql.